### PR TITLE
[ru] add missing metadata in `files/jsondata/`

### PR DESCRIPTION
**Pull request by leon-win**
### Description

This PR add missing metadata for `ru` locale in `files/jsondata/`

### Related issues and pull requests

Relates to https://github.com/mdn/translated-content/issues/8749

### CODE REVIEW

**Review by bsmth — APPROVED** [review]
Thank you! Leaving a +1, but maybe @pepelsbey would like to have a look 😄

**Review by pepelsbey — COMMENTED** [review]
Hey! Thanks for the fixes, they totally make sense :) A few suggestions, maybe.

File: files/jsondata/L10n-Common.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
     "ja": "実装先:",
     "ko": "구현:",
     "pt-BR": "Implementado por:",
-    "ru": "Выполняется:",
+    "ru": "Реализовано для:",
```
[the remark this answers is not in the document]
Comment (leon-win) [reply files/jsondata/L10n-Common.json]: Hello, @pepelsbey ! 
This translation is used in the following context (at the moment there are no such examples, so I created it artificially)):
Screenshot from 2023-12-11 16-24-55
Screenshot from 2023-12-11 16-25-01

That is, we are not talking about the capabilities of browsers, but about the implementation of interfaces by objects.

**Review by pepelsbey — APPROVED** [review]
LGTM :) Thank you!

**Comment by leon-win** [comment]
> Hey! Thanks for the fixes, they totally make sense :) A few suggestions, maybe.

Thank you for help! :+1:

**Review by bsmth — APPROVED** [review]
Thanks, both, merging now 🚢

**Pull request merged**